internal/wasm/interpreter/interpreter.go

```diff
@@ -188,57 +170,58 @@ type interpreterOp struct {
 	rs     []*wazeroir.InclusiveRange
 }
 
+// CompileModule implements the same method as documented on wasm.Engine.
+func (e *engine) CompileModule(module *wasm.Module) error {
+	if _, ok := e.getCompiledFunctions(module); ok { // cache hit!
+		return nil
+	}
+
+	funcs := make([]*compiledFunction, 0, len(module.FunctionSection))
+	if module.IsHostMdule() {
```

Comment:
maybe comment like "host modules are already compiled, so there's nothing to do... wasm functions need to be translated from wasm binary format into wazero's intermediate representation (wazeroir).